hw/ip/alert_handler/dv/cov/alert_handler_cov_excl.el

```diff
@@ -0,0 +1,26 @@
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: chencindy
+// Format Version: 2
+// Date: Fri Oct  9 13:59:32 2020
+// ExclMode: default
+//==================================================
+CHECKSUM: "1270015083 3167692650"
+INSTANCE: tb.dut
+Toggle tl_i.a_param "logic tl_i.a_param[2:0]"
+Toggle tl_o.d_user "logic tl_o.d_user[15:0]"
+Toggle tl_o.d_sink "logic tl_o.d_sink[0:0]"
+Toggle tl_o.d_param "logic tl_o.d_param[2:0]"
+Toggle tl_o.d_opcode [2] "logic tl_o.d_opcode[2:0]"
+Toggle tl_o.d_opcode [1] "logic tl_o.d_opcode[2:0]"
+Toggle tl_i.a_user.rsvd1 "logic tl_i.a_user.rsvd1[6:0]"
+Toggle tl_i.a_user.parity_en "logic tl_i.a_user.parity_en"
+Toggle tl_i.a_user.parity "logic tl_i.a_user.parity[7:0]"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [1][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [2][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [3][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [0][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
```

Comment:
These are newly excluded esc_cnt coverage. @msfschaffner we talked about this during an alert_handler DV meeting. Testbench will only cover up to 1000 cycles for each escalation phase length. Thus the esc_cnt can only go up to 1000 max.